modules/ROOT/pages/policy-mule3-add-remove-headers.adoc

```diff
@@ -4,79 +4,83 @@ include::_attributes.adoc[]
-
-== Downloading Add or Remove Header Policies
-
-Currently, the add or remove header policies are available on Anypoint Exchange. Follow the step-by-step procedures to download, set up, and apply these policies.
-
-To download the header policies:
-
-. Login to Anypoint Platform and go to Exchange.
-. Search for and download the policies, or use these URLs:
-+
-* Add Request Headers Policy
-+
-`+https://www.anypoint.mulesoft.com/exchange/?search=Add%20Request%20Headers%20Policy+`
-+
-* Remove Request Headers Policy
-+
-`+https://www.anypoint.mulesoft.com/exchange/?search=Remove%20Request%20Headers%20Policy+`
-+
-* Add Response Headers Policy
-+
-`+https://www.anypoint.mulesoft.com/exchange/?search=Add%20Response%20Headers%20Policy+`
-+
-* Remove Response Headers Policy
-+
-`+https://www.anypoint.mulesoft.com/exchange/?search=Remove%20Response%20Headers%20Policy+`
-+
-. Unzip each policy archive.
+For example, in the a request with header "foo=value1", if we add the header "foo=value2" with the policy, the result will be a request with header "foo=value1, value2"
 
-An XML and YAML file appears in each of the folders created by unzipping the archives. For example, the add-request-header-policy folder contains the following files:
+== Remove Request and Remove Response Headers
 
-* add-request.xml
-* add-request.yaml
+The Remove Request Header policy prevents receipt of one or more specified headers sent from the client to the backend service. The Remove Response Header policy prevents inclusion of one or more specified headers in a response from the backend service to the client.
 
-You use these files to set up the policy as described later.
-
-
-== Setting Up and Applying Header Policies
-
-If you have configured custom policies in Anypoint Platform, the process is basically the same for add or remove header policies. The following steps show how to set up and apply the policies in Anypoint Platform.
-
-To set up header policies:
+The prerequisites for using the header policies are:
 
-. Go to the main API Administration page.
-+
-. On the main menu drop-down, select Custom policies. (Top right corner)
-+
-Custom Policies appears listing custom policies, if there are any, that are available for the API.
-+
-. Click Add Custom Policy.
-. In Name, enter a name for the policy, for example `add request header policy`.
-. In Policy Definition, click Choose File to locate the YAML file, `add-request.yaml` in this example, that you downloaded.
-. In Policy Configuration, choose the XML file, for example, `add-request.xml`, that you downloaded.
-. Click Add.
-+
-image::gw-add-custom-pol.png[]
-+
-The policy, add request header policy in this example, appears on the Custom Policies page.
-+
-image::gw-custom-pol-list.png[]
+* Assume the role of Organization Administrators or API Versions Owner for an API version you want to manage in Anypoint Platform. Applying policies is an API version management task.
```

Comment:
To what text introduction do these list items apply?

Reply:
"The prerequisites for using the header policies are:" Now added a heading too.